src/watchmaker/exceptions/__init__.py

```diff
@@ -1,18 +1,127 @@
+import datetime
 import logging
+import os
 import sys
 
 
-def SystemFatal(msg):
-    """
-    This is a temporary exception handler for the project.
+class LogHandler:
```

Comment:
I agree .... I'll move it over to its own module and look into the other good suggestions as well.